source/exceptions.tex

```diff
@@ -114,10 +114,10 @@
   T1 t1;
   try {
     T2 t2;
-    if (@\textit{condition}@)
+    if (@\grammarterm{condition}@)
       goto lab;
-    } catch(...) { /* @\textit{handler 2}@ */ }
-  } catch(...) { /* @\textit{handler 1}@ */ }
+    } catch(...) { /* handler 2 */ }
```

Comment:
The comments should be in code font here. Have a look at the new `\commentellip` macro I made for inspiration on how to achieve this.